yast2_firewall_logging-level-20170210.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "yast2_firewall_logging-level"

```

Comment:
again, misleading tag name as the screenshot does not show a selected 'logging level'